### PR TITLE
create user 

### PR DESCRIPTION
**Pull request by aditisaxena259**
user created (CRUD op)

### CODE REVIEW

File: database/database.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"log"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
+
+func ConnectDatabase() {
+	dsn := "host=localhost user=postgres password=IIMB@1802 dbname=csi_rbac_system port=5432 sslmode=disable TimeZone=Asia/Kolkata"
```
Comment (TejasGhatte) [inline database/database.go:14]: database and migration file are present in initializers folder, so no need of rewrite. Also database credentials put in env.

File: main.go [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package main
```
Comment (TejasGhatte) [inline main.go:1]: why two main.go?

File: internal/routes/user_model.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	"github.com/google/uuid"
+)
+
+type User struct {
+	ID    uuid.UUID `json:"id"`
+	Name  string    `json:"name"`
+	Email string    `json:"email"`
+}
+
+func createResponseUser(userModel models.User) User {
```
Comment (TejasGhatte) [inline internal/routes/user_model.go:19]: no need of this function, send user variable directly in response

File: internal/routes/user_model.go [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+func CreateUser(c *fiber.Ctx) error {
+	var user models.User
+
+	if err := c.BodyParser(&user); err != nil {
+		return c.Status(http.StatusBadRequest).JSON(fiber.Map{
+			"error": "Invalid request body",
+		})
+	}
+
+	user.ID = uuid.New()
```
Comment (TejasGhatte) [inline internal/routes/user_model.go:37]: uuid and createdAt will be handled by postgres when a row is inserted, we dont have to explicitly assign them.

**Review by TejasGhatte — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (aditisaxena259) [reply main.go:1]: it was too test something, mb forgot to delete

**Pull request merged**